src/Engine/Manager/ParticleManager.cpp

```diff
@@ -61,13 +68,17 @@ void ParticleManager::UpdateBuffer(World& world) {
     particleRenderer->SetBufferContents(world.GetParticleCount(), world.GetParticles());
 }
 
-void ParticleManager::Render(World& world, const glm::vec3& position, const glm::vec3& up, const glm::mat4& viewProjectionMatrix, RenderSurface* renderSurface) {
-    if (world.GetParticleCount() > 0) {
-        UpdateBuffer(world);
-        particleRenderer->Render(textureAtlas, textureAtlasRowNumber, position, up, viewProjectionMatrix, renderSurface);
+void ParticleManager::UpdateParticleSystem(World& world, Component::ParticleSystemComponent* particleSystem) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'UpdateParticleSystem' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)